src/patient-registration/patient-registration.test.tsx

```diff
@@ -77,3 +78,97 @@ describe('contact info section', () => {
   updateAddress('country');
   updateAddress('postalCode');
 });
+
+describe('contact person section', () => {
```

Comment:
Looks good to me. We may need another ticket to ensure that all our section/input tests are consistent in the way that the tests are setup and structured.